cmd/build/helmify/replacements.go

```diff
@@ -123,6 +123,15 @@ var replacements = map[string]string{
     - '*'
   {{- end }}`,
 
+	"HELMSUBST_MUTATING_WEBHOOK_CLIENT_CONFIG: \"\"": `{{- if .Values.webhookURL }}
+    url: https://{{ .Values.webhookURL }}/v1/mutate
```

Comment:
perhaps different values for `validatingWebhookURL` and `mutatingWebhookURL` in case someone needs them to be separate?

Reply:
I've made the change to support those two URLs separately.